hibernate-envers/src/main/java/org/hibernate/envers/configuration/internal/metadata/CollectionMetadataGenerator.java

```diff
@@ -215,7 +217,8 @@ private void addOneToManyAttached(boolean fakeOneToManyBidirectional) {
 		// in a join table, so the prefix value is arbitrary).
 		final MiddleIdData referencedIdData = createMiddleIdData(
 				referencedIdMapping,
-				null, referencedEntityName
+				mappedBy + "_",
```

Comment:
not sure why this is needed, tried to change it back to null and all the tests passed